ntoskrnl/cache/mdlsup.c

```diff
@@ -83,7 +83,7 @@ NTAPI
 CcMdlReadComplete2(IN PFILE_OBJECT FileObject,
                    IN PMDL MdlChain)
 {
-    UNIMPLEMENTED
+	CcMdlReadComplete(FileObject, MdlChain);
```

Comment:
This is acceptable as a stub, but note that the current Cc has a correct implementation for all four of these.

Reply:
The current Cc doesn't seem to have correct implementations I'm afraid...

Reply:
They're much closer to correct than NewCc's.